### DISCUSSION TITLE
Como você se mantem atualizado sobre as notícias e novidades sobre tech?

### DISCUSSION THREAD

**Discussion by wilsonneto-swe** [frontendbr/forum#1991] [Discussão]
Fala pessoal, beleza?

Em nossa área a todo momento tem releases de techs e frameworks, notícias, falhas de segurança, novas techs, empresas mudando suas estratégias, aquisições, novos padrões de mercado... Enfim, é uma tarefinha complicada se manter atualizado de tudo rs

Como vocês fazem para se manterem atualizados sobre isso tudo? Algum site, newsletter, portal, fórum, notificações do GH, o que você usa?

E se alguém se interessar, como eu trabalho com JS/ES/TS, Node.js, ReactJS e NextJS, e preciso me manter informado diáriamente sobre estes assuntos, criei uma newsletter onde junto tudo que realmente é importante do que vi durante a semana de notícias e novidades, e envio ali semanalmente, se alguém curtir a idéia e quiser se inscrever: [link da newsletter](https://novidadesjs.substack.com/welcome?utm_source=frontbr)

Atualmente como você se mantem informado?

Forte abraço galera!

**Comment by felipefialho** [comment]
Um tempinho atrás escrevi esse [texto contando das estratégias que uso pra me atualizar](https://www.felipefialho.com/blog/como-se-atualizar-sobre-desenvolvimento-front-end/), não mudou muito desde então 😁

**Comment by ninetails** [comment]
Humm, o post do Felipe Fialho já fala tudo ;D

Não utilizo muito o Twitter, mas seguindo os gringos ajuda bastante a saber o que tá rolando de novidades praticamente saindo do forno rs.

Complementando:

- **Grupos no Slack ou Telegram**, tem vários grupos ativos dos mais variados assuntos, e a galera sempre posta umas novidades no meio das conversas;
- **Meetups** não chegam a ser "novidades" mas sempre vai ter aquela palestra que alguém manda algo que vc não chegou a ver anteriormente ou só estava no radar mas nunca deu atenção;
- Pela mesma razão acima, ver no **YouTube**, as _streams_ de eventos gringos;
- Também vejo uma galera começando a utilizar o **LinkedIn** como alternativa ao Medium/dev.to.

**Comment by darthrodolfo** [comment]
Utilizo site da _Medium_, _Wired_, _Twitter_ e canais do _Youtube_ voltados pra Dev.

E tambem to procurando adicionar mais canais de informação, o _Linkedin_ tem uma rotatividade interessante de noticias e novidades do mundo da programação. Lives realizadas por comunidades ou canais do Youtube tambem tem agregado bastante. E o forum aqui do Github do FrontEndBR.

Utilizo aplicativos e tambem recebo por e-mail diversos links interessantes para esses portais! O medium por exemplo ja tras listas de leituras personalidades com artigos que talvez me interessem! É fantástico hahaha.

**Comment by willianjusten** [comment]
Eu tenho utilizado bastante o [Daily.dev](https://daily.dev/) numa extensão de nova guia. Aí toda vez que abro uma guia nova, tem notícias lá e talz. Tu pode adicionar canais de áreas que se interessa, separar por categorias e etc.

**Comment by rhuangabrielsantos** [comment]
Olá amigos!

Eu gosto muito de utilizar o https://tympanus.net/codrops/, eles possuem uma "revista" semanal com coisas muito interessantes sobre o mundo do frontend, além do citado Daily.dev que é muito bom!

Abraços!

**Comment by glauberm** [comment]
Como ninguém mencionou ainda: vale também acompanhar o [Hacker News](https://news.ycombinator.com/), rolam boas discussões por lá.

**Comment by YasmimVieira** [comment]
Eu uso uma extensão chamada daily.dev que me mostra vários conteúdos novo assim que abro uma página no google, além disso eu também gosto de seguir bastante influencers da área que postam conteúdos interessantes e também fico fuçando forum e artigos no medium e documentações.

**Comment by ojvribeiro** [comment]
Eu recomendo demais a Newsletter do @filipedeschamps. As notícias são bem resumidas e vão direto ao ponto. Ideal pra quem não tem muito tempo de ficar lendo notícias.

O link tá aqui: https://filipedeschamps.com.br/newsletter
